modules/installation-creating-gcp-worker.adoc

```diff
@@ -39,11 +43,31 @@ have to contact Red Hat support with your installation logs.
 section of this topic and save it as `06_worker.py` on your computer. This
 template describes the worker machines that your cluster requires.
 
-. Export the following variables needed by the resource definition:
+. Export the variables that the resource definition uses.
+.. Export the subnet that hosts the compute machines:
 +
+ifndef::shared-vpc[]
 ----
 $ export COMPUTE_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-worker-subnet --region=${REGION} --format json | jq -r .selfLink`
```

Comment:
Need to specify --project --account:
export COMPUTE_SUBNET=$(gcloud compute networks subnets describe <worker-subnet> --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)

Reply:
I'm sorry that I forgot an option after describe. The correct command line is like this:
export COMPUTE_SUBNET=$(gcloud compute networks subnets describe **${HOST_PROJECT_COMPUTE_SUBNET}** --region=${R    EGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)

Reply:
Because of the if statement format, this change should show up on line 56.